src/constants/functionKey.ts

```diff
@@ -4,7 +4,6 @@ enum FunctionKey {
   GET_MARKETS = 'GET_MARKETS',
   GET_MARKET_HISTORY = 'GET_MARKET_HISTORY',
   GET_ASSETS_IN_ACCOUNT = 'GET_ASSETS_IN_ACCOUNT',
-  GET_HYPOTHETICAL_LIQUIDITY = 'GET_HYPOTHETICAL_LIQUIDITY',
```

Comment:
Don't we still need this for the query?

Reply:
No because we no longer use React Query to call `getHypotheticalAccountLiquidity`